src/main/java/seedu/address/logic/parser/TimeParserUtil.java

```diff
@@ -65,6 +70,10 @@
     public static LocalTime parse(String timeString) throws ParseException {
         requireNonNull(timeString);
 
+        if (timeString.trim().equals("")) {
+            throw new ParseException(MESSAGE_EMPTY_TIME);
+        }
+
```

Comment:
Just some Java trivia.  
You can also take a look at the isBlank() method for String. It returns true if the string is empty or only contain trailing whitespaces. 😄

Reply:
thanks for the advice!